tpm2/tpm2_test.go

```diff
@@ -16,21 +16,41 @@ package tpm2
 
 import (
 	"bytes"
+	"context"
 	"crypto"
 	"crypto/ecdsa"
 	"crypto/elliptic"
 	"crypto/rand"
 	"crypto/rsa"
 	"crypto/sha256"
 	"flag"
+	"io"
 	"math/big"
 	"os"
 	"reflect"
 	"testing"
 
 	"github.com/google/go-tpm/tpmutil"
+	"github.com/google/go-tpm/tpmutil/mssim"
 )
 
+var tpmMSSIM = flag.Bool("tpm_mssim", false, "Run integration tests against a Microsoft simulator.")
```

Comment:
Expose the Platform and Command ports instead

Reply:
done